### ISSUE TITLE
Activity Maintaining

### ISSUE THREAD

**Issue by t-kruth** [hyperjumptech/grule-rule-engine#463]
Hello!,

I created a discussion a few weeks back https://github.com/hyperjumptech/grule-rule-engine/discussions/460

I was curious of the status of this project, and if maintainers/community is active. it's very appealing but the lack of recent activity is a little alarming

**Comment by knidarkness** [comment]
Also am curious as to current status of the project.

**Comment by chbomtempo** [comment]
Is there a way to contact hyperjumptech ? They claim to be a company, but I do not found a single way to contact them other than maybe job@hyperjump.tech... Has anyone tried to contact an employee? https://www.linkedin.com/company/hyperjump/people/

**Comment by chbomtempo** [comment]
Is there a way to contact hyperjumptech ? They claim to be a company, but I do not found a single way to contact them other than maybe job@hyperjump.tech... Has anyone tried to contact an employee? https://www.linkedin.com/company/hyperjump/people/
Maybe @newm4n ?

**Comment by t-kruth** [comment]
> Is there a way to contact hyperjumptech ? They claim to be a company, but I do not found a single way to contact them other than maybe [job@hyperjump.tech](mailto:job@hyperjump.tech)... Has anyone tried to contact an employee? https://www.linkedin.com/company/hyperjump/people/ Maybe @newm4n ?

@chbomtempo I this was my attempt at reaching out. I haven't tried any other avenues. If you do, please let us know here!

**Comment by 5idu** [comment]
I have forked this repository and currently have three new features.
- [fix: load rules order](https://github.com/hyperjumptech/grule-rule-engine/commit/7e7054732f68989ea19f2475b5edd2df60b7b05d)
- [upgrade antlr package version](https://github.com/hyperjumptech/grule-rule-engine/commit/57039ffed37621f175ffb939d2e48cc978cfd162)

Contributions welcome！

**Comment by jonaswouters** [comment]
I think it would be better to make a new open organisation to put the fork under, and have some participants of this ticket be the maintainers?

**Comment by stampthecoder** [comment]
@jonaswouters  @5idu I agree! Can we start one. I would like to be a contributor. I can start one if needed.

**Comment by jonaswouters** [comment]
> @jonaswouters @5idu I agree! Can we start one. I would like to be a contributor. I can start one if needed.

go ahead 👍

**Comment by mts57017** [comment]
would love to help as well, please add me

**Comment by ralphg6** [comment]
Add me too..

**Comment by Jonfor** [comment]
Hey guys, I'm also a user of Grule that is looking into forking this. I can only do light contributions like updating the go version and dependencies. Is there a new organization fork yet? I only see @5idu 's fork with [a few commits](https://github.com/5idu/grule-rule-engine).

**Comment by jonaswouters** [comment]
I was about to mail @stampthecoder about it, who said he was willing to set it up. 
We just have to align that somebody does it, and not end up with 3 different ones :)

**Comment by stampthecoder** [comment]
@jonaswouters @Jonfor @ralphg6 @chbomtempo Sent out invites to new org!

**Comment by vijaykramesh** [comment]
@stampthecoder could I get an invite as well? we actively use grule-rule-engine in production, are considering deprecating it due to lack of development/maintenance but I'd much rather join in with an active fork!

**Comment by stampthecoder** [comment]
Absolutely!

**Comment by jonaswouters** [comment]
> Is there a way to contact hyperjumptech ? They claim to be a company, but I do not found a single way to contact them other than maybe [job@hyperjump.tech](mailto:job@hyperjump.tech)... Has anyone tried to contact an employee? https://www.linkedin.com/company/hyperjump/people/ Maybe [@newm4n](https://github.com/newm4n) ?

I tried reaching out a while ago, never got a response.

Now we need some momentum for [the fork](https://github.com/DataWiseHQ/grule-rule-engine), and a better way to communicate (I propose discord)?

**Comment by ralphg6** [comment]
@stampthecoder, could we enable the Issues in the new repo? 

Because it can be a place to open the new discussions...  

Only for knowledge, our new repo is https://github.com/DataWiseHQ/grule-rule-engine

**Comment by stampthecoder** [comment]
Absolutely! I’ll do that today

**Comment by ralphg6** [comment]
@jonaswouters @Jonfor @stampthecoder  @chbomtempo @vijaykramesh 

What do you think about copying all current PRs to the new repo? 

We could evaluate which one we would merge with!

I could try to use the GitHub API to do it.

**Comment by ralphg6** [comment]
Maybe... A good idea is to copy the issues too..

**Comment by stampthecoder** [comment]
That sounds like a great idea. Maybe we can think of a cicd pipeline. I set
up a basic one that will run a build and run some unit tests. We can add
security checks which would align to corporate values and lead to better
adoption.

**Comment by ralphg6** [comment]
Yes... A pipeline to copy the issues and another one to copy the PRs is a good idea.. 

I will create it on the new repo... 

Soon, I will create a PR with it.

**Comment by ralphg6** [comment]
> Absolutely! I’ll do that today
> […](#)

![Image](https://github.com/user-attachments/assets/0e13c267-fad8-446f-8a08-d9b4da8957ee)

Please, enable the Issues and Discussions..

**Comment by stampthecoder** [comment]
Done.

**Comment by t-kruth** [comment]
I'll def try to do some contributions to the new fork etc. Love seeing the community come together

**Comment by newm4n** [comment]
Hi All, first of all, I would like to say THANK YOU ALL.
Its all been my short comming. Mainly due to my health issue. But because of all of your invaluable trust on the project, the project
keeps going alive, through continuous contributions and/or forks.

I'm fully aware of my limitation and thus looking for good watcher and caretaker of this open source project, so it wont goes vacum like it was.
I will fully help you if you're willing to contribute your time on the project, as long as I have the user right to do so in github, and under the organization, hyperjump. Eg. making new reviewer, grule admins, releaser, etc.

In my current state, what I've been doing so far is.

- Comment or Accept or Reject PRs.
- Auto Merge new changes whenever possible if some commits in the master is ahead of the PR, If can't. I will comment to ask the PR  maker to update their branch.
- Approve the workflow  to run, if needed or not done automatically by github.
- Make a tag before release.
- Make a new release.

I have a good hope that you will help to do any of those and I willing to help to enable you to do it. 
Lets discuss this or advise me to do next appropriately.

Ferdinand
